<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_COAS_StatePart</class>
 <widget class="QWidget" name="P_COAS_StatePart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3,4">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>11</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">
font-family:&quot;微软雅黑&quot;;
font-size:20px;
font-weight:bold;
padding:4px;
color:#ffffff;
border-radius:3px;
border:1px solid #000000;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #444444, stop:0.3 #525252, stop:0.9 #525252, stop:1 #444444 )</string>
              </property>
              <property name="text">
               <string>状态元</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="styleSheet">
               <string notr="true">
QTableView::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #5C87D9, stop:0.505 #557CC7, stop:1 #5075BC);
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_param">
              <property name="title">
               <string>参数</string>
              </property>
              <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
               <item row="4" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>状态元权重</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QSpinBox" name="spinBox_proportion">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_name"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>状态元标签</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_tagTank">
                 <property name="text">
                  <string>[]</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>状态元优先级</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButton_tagTank">
                 <property name="text">
                  <string>编辑标签</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>状态元名称</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QCheckBox" name="checkBox_gif_back_run">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="9" column="1">
                <widget class="QCheckBox" name="checkBox_smooth">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>是否预加载</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QCheckBox" name="checkBox_gif_preload">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>图像-色调值</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>图像-模糊边缘</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="1">
                <widget class="QPlainTextEdit" name="plainTextEdit_note">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="10" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>备注</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QSlider" name="horizontalSlider_tint">
                 <property name="maximum">
                  <number>360</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="spinBox_priority">
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>是否倒放</string>
                 </property>
                </widget>
               </item>
               <item row="11" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>9</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>可被动作元打断</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QCheckBox" name="checkBox_canBeInterrupted">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_animPlayer">
              <property name="title">
               <string>动画帧</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_view">
           <property name="title">
            <string>视图</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <property name="styleSheet">
               <string notr="true">
QToolButton{
	padding:2px;
	border: 1px solid transparent;
	background: rgba(255,255,255,0);
}
QToolButton:hover{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #fdecad, stop:0.3 #fde48c, stop:0.9 #fce58c, stop:1 #fdfadf )
}
QToolButton:pressed{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="toolButton_zoom_in">
                 <property name="toolTip">
                  <string>放大</string>
                 </property>
                 <property name="text">
                  <string>放大</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../../drillgifactionsequenceeditor.qrc">
                   <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_In.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_In.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_zoom_out">
                 <property name="toolTip">
                  <string>缩小</string>
                 </property>
                 <property name="text">
                  <string>缩小</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../../drillgifactionsequenceeditor.qrc">
                   <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Out.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Out.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_zoom_regular">
                 <property name="toolTip">
                  <string>原比例</string>
                 </property>
                 <property name="text">
                  <string>原比例</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../../drillgifactionsequenceeditor.qrc">
                   <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Regular.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Regular.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_zoomValue">
                 <property name="text">
                  <string>100.0%</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_view" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">
QListView::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #5C87D9, stop:0.505 #557CC7, stop:1 #5075BC);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../drillgifactionsequenceeditor.qrc"/>
 </resources>
 <connections/>
</ui>
